src/contracts/misc/GhoGsmSteward.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {EnumerableSet} from '@openzeppelin/contracts/utils/structs/EnumerableSet.sol';
+import {FixedFeeStrategy} from '../facilitators/gsm/feeStrategy/FixedFeeStrategy.sol';
+import {IGsm} from '../facilitators/gsm/interfaces/IGsm.sol';
+import {IGsmFeeStrategy} from '../facilitators/gsm/feeStrategy/interfaces/IGsmFeeStrategy.sol';
+import {IGhoGsmSteward} from './interfaces/IGhoGsmSteward.sol';
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+
+/**
+ * @title GhoGsmSteward
+ * @author Aave Labs
+ * @notice Helper contract for managing parameters of the GSM
+ * @dev Only the Risk Council is able to action contract's functions, based on specific conditions that have been agreed upon with the community.
+ * @dev Requires role GSM_CONFIGURATOR_ROLE on the GhoGsm contract and CONFIGURATOR on every GSM asset to be managed
+ */
+contract GhoGsmSteward is Ownable, RiskCouncilControlled, IGhoGsmSteward {
+  using EnumerableSet for EnumerableSet.AddressSet;
+
+  /// @inheritdoc IGhoGsmSteward
+  uint256 public constant GSM_FEE_RATE_CHANGE_MAX = 0.0050e4; // 0.50%
+
+  /// @inheritdoc IGhoGsmSteward
+  uint256 public constant MINIMUM_DELAY = 2 days;
+
+  /// @inheritdoc IGhoGsmSteward
+  function RISK_COUNCIL() public view override returns (address) {
+    return COUNCIL;
+  }
+
+  mapping(address => GsmDebounce) internal _gsmTimelocksByAddress;
+
+  mapping(uint256 => mapping(uint256 => address)) internal _gsmFeeStrategiesByRates;
```

Comment:
can we have a separate registry contract for this so it can be reused?

Reply:
Will look into how new borrow rate updates happen now